src/library/basesqltablemodel.cpp

```diff
@@ -732,7 +735,7 @@ QVariant BaseSqlTableModel::data(const QModelIndex& index, int role) const {
         QColor color = mixxx::RgbColor::toQColor(
                 mixxx::RgbColor::fromQVariant(getBaseValue(colorIndex, role)));
         if (color.isValid()) {
-            color.setAlpha(32); // Make this 12.5% opaque
+            color.setAlpha(kTrackColorRowBackgroundOpacity);
             value = QBrush(color);
         }
```

Comment:
`else value = QVariant()`

Otherwise the value contains a valid value, but not a QBrush. Please always compile with `-DDEBUG_ASSERTIONS_FATAL=ON`!

We should also think to avoid calling `value = getBaseValue(index, role);` because the value is discarded immediately.

Reply:
Qt::BackgroundRole should be handled as a special case before all other roles that depend on getBaseValue(). It could even be handled before the role whitelist `if`.

Reply:
I need to extract this part for `BaseTrackTableModel` anyway, so let's just keep it as it is now.